src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/TriviaEngine/LineColumnRule.cs

```diff
@@ -124,9 +124,9 @@ public static LineColumnRule ForceSpacesOrUseAbsoluteIndentation(int spacesOrInd
             => new(
                 SpaceOperations.Force,
                 LineOperations.Preserve,
-                IndentationOperations.Absolute,
+                IndentationOperations.Follow,
```

Comment:
i don't understand this.  The method name is ForceSpacesOrUseAbsoluteIndentation, but you don't set Absolute.  You set 'Follow'.

Reply:
Renamed